h5py/h5t.pyx

```diff
@@ -764,7 +766,39 @@ cdef class TypeBitfieldID(TypeID):
     """
         HDF5 bitfield type
     """
-    pass
+    @with_phil
+    def get_sign(self):
+        """() => INT sign
+
+        Get the "signedness" of the datatype; one of:
+
+        SGN_NONE
+            Unsigned
+
+        SGN_2
+            Signed 2's complement
+        """
+
+        # This will always return SGN_NONE to cast bitfield types to unsigned integer
+        return SGN_NONE
```

Comment:
I don't think we should have a method for this (the relevant HDF5 method [is documented as](https://confluence.hdfgroup.org/display/HDF5/H5T_GET_SIGN) "for an integer type"). We can just hardcode the `'u'` for unsigned in `py_dtype`.

Reply:
Yep okay. I do return a constant `SGN_NONE` so that it will always be `'u'`- I thought this might be more "consistent" with the rest of the integer implementation, but I can see how it might just be confusing. I'll change it to just be `'u'`.